recipes/inermat/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "intermat" %}
+{% set version = "2024.3.24" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/usnistgov/intermat/archive/refs/tags/v2024.3.24.tar.gz
```

We need that for the autoupdates.

```suggestion
  url: https://github.com/usnistgov/intermat/archive/refs/tags/v{{ version }}.tar.gz
```